lib/portus/migrate.rb

```diff
@@ -14,29 +14,37 @@ module Migrate
     #
     # TODO: (mssola) remove in the next version.
     def self.from_humanized_time(duration, default)
-      return duration.send(:minutes) if duration.is_a? Integer
+      return duration.minutes if duration.respond_to? :minutes
 
       # If it's not a String then just return the given default value.
-      return default.send(:minutes) unless duration.is_a? String
+      return default.minutes unless duration.is_a? String
 
-      # If it's a string containing just a number, then convert it and return
-      # it. Otherwise, if it has a bad (old) format, then just return the default.
-      splitted = duration.split(".")
-      if splitted.size == 1
-        val = splitted.first.to_i
-        return val.send(:minutes) unless val == 0
-      end
-      return default.send(:minutes) if splitted.size != 2
+      # If we got an empty String then return the default value.
+      return default.minutes if duration.empty?
 
-      Rails.logger.tagged("deprecated") do
-        Rails.logger.warn "The 'x.minutes' format is deprecated for configuration values such " \
-                          "as `jwt_expiration_time`. From now on these values are expected to " \
-                          "be integers representing minutes."
+      raw_value, method = duration.split('.')
+      value = raw_value.to_i
+
+      # When we found a callable method, notify about deprecation
+      unless method.nil?
+        if value.respond_to? method
```

Comment:
rubocop will complain about this (it doesn't now because CI is broken, working on that...). Instead you should use a guard clause:

```ruby
return unless value.respond_to? method
```